File: src/test/java/org/openrewrite/python/tree/ReturnTest.java
package org.openrewrite.python.tree;

import org.junit.jupiter.api.Test;
import org.openrewrite.test.RewriteTest;

import static org.openrewrite.python.Assertions.python;

public class ReturnTest implements RewriteTest {

    @Test
    void methodReturn() {
        rewriteRun(
          python(
            """
              def foo():
                  return 1
              """
          )
        );
    }
}

File: rewrite/tests/python/return_test.py
from rewrite.test import rewrite_run, python


def test_empty():
    # language=python
    rewrite_run(
        python(
            """
            def foo():
                return
            """
        )
    )


def test_trailing_semicolon():
    # language=python
    rewrite_run(
        python(
            """
            def foo():
                return;
            """
        )
    )


def test_value():
    # language=python
    rewrite_run(
        python(
            """
            def foo():
                return 1
            """
        )
    )
